doc/introduction/measurements.rst

```diff
@@ -145,3 +175,44 @@ number of shots. This can be done by modifying the value of :attr:`.Device.shots
     # execute the QNode again, now using 1 shot
     dev.shots = 1
     result = circuit(0.54, 0.1)
+
+
+With an increasing number of shots, the average over
+measurement samples converges to the exact expectation of an observable. Consider the following
+circuit:
+
+.. code-block:: python
+
+    from pennylane import numpy as np
+
+    # fix seed to produce same samples in different runs
+    np.random.seed(1)
+
+    dev = qml.device("default.qubit", wires=1)
+
+    @qml.qnode(dev)
+    def circuit():
+        qml.Hadamard(wires=0)
+        return qml.expval(qml.PauliZ(0))
+
+Running the simulator without specifying the shot number returns the exact expectation.
```

Comment:
```suggestion
Running the simulator when the ``analytic`` is set to ``True`` returns the exact expectation.
```

Reply:
The reason for this suggestion is that ``shots=1000`` is default (e.g. for ``default.qubit``), so for ``dev = qml.device("default.qubit", analytic=False, wires=1)`` one would get results based on a ``1000`` samples